docs/designs/2686-proposal-process.md

```diff
@@ -0,0 +1,90 @@
+# Proposing Changes to Tanzu Community Edition
+
+* Proposal: [https://github.com/vmware-tanzu/community-edition/issues/2686](https://github.com/vmware-tanzu/community-edition/issues/2686)
+
+## Introduction
+
+Tanzu Community Edition relies on design proposals to assess the viability of non-trivial changes
+to the project. This document covers details on the proposal process, which can also be used as
+guidelines for how to create a proposal.
+
+## The Proposal Process
+
+The proposal process is the process for reviewing a proposal and reaching a decision about whether
+to accept or decline the proposal.
+
+1. The proposal author [creates a brief issue](https://golang.org/issue/new) describing the proposal.
+
+   > A design doc is **not** required at this point.
```

Comment:
Maybe soften this with saying a design doc is welcome if they want to provide one, but not required at this point.

Reply:
Agreed.

Also: we should encourage that POCs and code **are** ok! You don't need to wait. But you should know that you're running the risk of the proposal getting declined and the code going to waste.

Reply:
Added verbiage to soften this and also talk a bit about starting an implementation before a proposal is accepted.